srv/src/main/resources/application.yaml

```diff
@@ -18,6 +18,8 @@ cds:
         - "10401010"
     - name: user
       password: user
+  features:
+    assert_integrity: db
```

Comment:
This is not a valid configuration in application.yaml.

Reply:
Let's skip the actual assertions on db level for now. I think it might be valuable to showcase this here in the bookshop, but I guess this is a bigger change..

Reply:
cdsrc.json?